### PR TITLE
[BFN] Fixed SONiC fwutil exec time

### PR DESCRIPTION
**Pull request by tkerykx**
Signed-off-by: Taras Keryk <tarasx.keryk@intel.com>

#### Why I did it
The execution time of the SONiC fwutil command is more than 30s if the BFN SDK is not updated to the latest version
#### How I did it
Added a wrapper to platform_utils to limit function execution time
#### How to verify it
The execution time of the SONiC fwutil command  without  updating SDK to the latest version should be a maximum of a few seconds
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

### CODE REVIEW

**Review by akokhan — DISMISSED** [review]

**Comment by tkerykx** [comment]
@prgeor, can you please review

**Comment by tkerykx** [comment]
@prgeor, can you please review

**Comment by tkerykx** [comment]
@prgeor, can you please review

**Comment by tkerykx** [comment]
@prgeor, can you please review
Thanks!

File: platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/platform_utils.py [hunk 1]
```diff
@@ -18,5 +26,33 @@ def run_cmd(cmd):
         run_cmd(['mkdir', '-p', file_path])
     if not os.path.isfile(path):
         run_cmd(['touch', path])
-    if (mode is not None):    
+    if (mode is not None):
         run_cmd(['chmod', mode, path])
+
+def limit_execution_time(execution_time_secs: int):
```
Comment (akokhan) [inline platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/platform_utils.py:32]: I'd expect the first param should be `func`
Please check

Comment (akokhan) [inline platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/platform_utils.py:32]: I'd expect the first param should be `func`
Please check

Comment (tkerykx) [reply platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/platform_utils.py:32]: I checked, and I see, for decorators with arguments it is recommended to use func as param in the inner function. If you don't mind

Comment (tkerykx) [reply platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/platform_utils.py:32]: I checked, and I see, for decorators with arguments it is recommended to use func as param in the inner function. If you don't mind

**Pull request closed without merging**